.common-ci.yml

```diff
@@ -81,6 +81,12 @@ trigger-pipeline:
     matrix:
       - DRIVER_VERSION: [535.216.01, 550.127.05]
 
+# Define the driver versions for jobs that can be run in parallel for amzn2023
+.driver-versions-amzn2023:
+  parallel:
+    matrix:
+      - DRIVER_VERSION: [560.35.03]
```

Comment:
Why only R560? Can we not build R550 ? R550 is our default default driver branch

Reply:
added

Reply:
550 support package is not available  
https://developer.download.nvidia.com/compute/cuda/repos/amzn2023/x86_64/  (supports 560,555)